src/editor.rs

```diff
@@ -767,7 +771,7 @@ mod tests {
     fn delete_all_after_cursor_undo() {
         let mut context = Context::new();
         let out = Vec::new();
-        let mut ed = Editor::new(out, "prompt".to_owned(), &mut context).unwrap();
+        let mut ed = Editor::new(out, "prompt".to_owned(), |s| {s.to_string()}, &mut context).unwrap();
```

Comment:
Consider using `String::from` instead of `|s| {s.to_string()}` :)

Reply:
Done.

Reply:
Oh no, I meant literally `String::from`. As in `Editor::new(out, "prompt".to_owned(), String::from, &mut context)` :)